packages/core/src/components/table/table-body-row-expandable/table-body-row-expandable.tsx

```diff
@@ -101,6 +104,14 @@ export class TdsTableBodyRowExpandable {
     }
   }
 
+  @Listen('tdsChange', { target: 'body' })
+  handleRowExpand(event: CustomEvent<{ rowId: string; isExpanded: boolean }>) {
+    /** Collapse all other rows when autoCollapse is true and a row is expanded */
+    if (this.autoCollapse && event.detail.isExpanded && event.detail.rowId !== this.rowId) {
```

Comment:
Might wanna check that the tableId is the same as `this.tableId` aswell. Given that a page could have multiple tables and in theory the id of the rows should be shared between tables. For example if a user uses index as the id.

Reply:
Good point which I did not think of, thanks!